src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/Models/Manifest.cs

```diff
@@ -103,4 +106,69 @@ public class Blob
         [XmlAttribute(AttributeName = "NonShipping")]
         public bool NonShipping { get; set; }
     }
+
+    [XmlRoot(ElementName = "SigningInformation")]
+    public class SigningInformation
+    {
+        [XmlAttribute(AttributeName = "AzureDevOpsCollectionUri")]
+        public string AzureDevOpsCollectionUri { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsProject")]
+        public string AzureDevOpsProject { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsBuildId")]
+        public string AzureDevOpsBuildId { get; set; }
+
+        [XmlElement(ElementName = "FileExtensionSignInfo")]
+        public List<FileExtensionSignInfo> FileExtensionSignInfos { get; set; }
+
+        [XmlElement(ElementName = "FileSignInfo")]
+        public List<FileSignInfo> FileSignInfos { get; set; }
+
+        [XmlElement(ElementName = "ItemsToSign")]
+        public List<ItemsToSign> ItemsToSign { get; set; }
+
+        [XmlElement(ElementName = "StrongNameSignInfo")]
+        public List<StrongNameSignInfo> StrongNameSignInfos { get; set; }
+    }
+
+    [XmlRoot(ElementName = "FileExtensionSignInfo")]
+    public class FileExtensionSignInfo
+    {
+        [XmlAttribute(AttributeName = "Extension")]
+        public string Extension { get; set; }
+
+        [XmlAttribute(AttributeName = "CertificateName")]
+        public string CertificateName { get; set; }
+    }
+
+    [XmlRoot(ElementName = "FileSignInfo")]
+    public class FileSignInfo
+    {
+        [XmlAttribute(AttributeName = "File")]
+        public string File { get; set; }
```

Comment:
Is this supposed to be the full path? just the name? Would it make sense to have a File model that includes more information?

Reply:
Same comment everywhere File is a string.

Reply:
It juts represents the name of the file we'll sign

Reply:
Okay so we assume that foo.dll is basically the same everywhere

Reply:
I think that to be the case. @jonfortescue can you confirm?

Reply:
Yep, exactly, as per [Signing.props in arcade](https://github.com/dotnet/arcade/blob/master/eng/Signing.props#L7-L29).